src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -112,14 +127,23 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example to allow one single parameterless annotation on the same line
+ * Example to allow multiple annotations on the same line:
  * </p>
  * <pre>
- * &#64;Override public int hashCode() { ... } // no violations
- * &#64;SuppressWarnings("deprecation") public int foo() { ... } // violation
+ * &#64;NotNull private boolean field1; //ok
+ * &#64;Override public int hashCode() { return 1; } //ok
+ * &#64;NotNull //ok
+ * private boolean field2;
+ * &#64;Override //ok
+ * public boolean equals(Object obj) { return true; }
+ * &#64;Mock DataLoader loader; //ok
+ * &#64;SuppressWarnings("deprecation") DataLoader loader; //ok
+ * &#64;SuppressWarnings("deprecation") public int foo() { return 1; } //ok
+ * &#64;SuppressWarnings("deprecation") &#64;Mock DataLoader loader; //ok
  * </pre>
  * <p>
- * Use the following configuration:
+ * Use the following configuration to allow one single parameterless
+ * annotation on the same line.This configuration is the same as default configuration:
```

Comment:
we do not need this, as it is same as default, just move description to default block.

Reply:
The way of defining the configurations for the 2 instances are different. So is it okay to move both configurations into one!!

Reply:
yes, it is ok squash them in one, into default form, make description like `Default configuration, to allow one single parameterless annotation on the same line:`
in other words ... this config+example is not required.

Reply:
Do I need to show only the default config or both cofigs under the description ```Default configuration, to allow one single parameterless annotation on the same line:```?

Reply:
only default config please